packages/dcos-integration-test/extra/test_packaging.py

```diff
@@ -149,3 +148,23 @@ def test_packaging_api(dcos_api_session):
     list_response = dcos_api_session.cosmos.list_packages()
     packages = list_response.json()['packages']
     assert len(packages) == 0
+
+
+def test_mom_installation(dcos_api_session):
+    """Test the Cosmos installation of marathon on marathon (MoM)
+    """
+
+    install_response = dcos_api_session.cosmos.install_package('marathon')
+    data = install_response.json()
+
+    dcos_api_session.marathon.wait_for_deployments_complete()
```

Comment:
Would be nice to confirm that the deployment won't complete until health checks pass. I'm pretty sure that is the case :)